spec/integ/matrix-client-event-timeline.spec.ts

```diff
@@ -1091,6 +1180,9 @@ describe("MatrixClient event timelines", function () {
                     event_id: THREAD_ROOT.event_id,
                 },
             },
+            unsigned: {
+                [UNSIGNED_THREAD_ID_FIELD.name]: THREAD_ROOT.event_id,
+            },
```

Comment:
I'm somewhat concerned that by changing all these, we never test the fallback behaviour. Do we have any separate tests for the fallback? Do we actually need to change all these definitions?